consensus/parlia/parlia.go

```diff
@@ -426,11 +592,22 @@ func (p *Parlia) verifyCascadingFields(chain consensus.ChainHeaderReader, header
 		return fmt.Errorf("invalid gas limit: have %d, want %d += %d", header.GasLimit, parent.GasLimit, limit)
 	}
 
+	// Verify vote attestation for fast finality.
+	if err := p.verifyVoteAttestation(chain, header, parents); err != nil {
+		if chain.Config().IsLynn(header.Number) {
+			return err
+		}
+		log.Warn("Verify vote attestation failed", "block", header.Number.Uint64(), "block hash", header.Hash(), "error", err)
```

Comment:
Can you log the whole header here? It will help to locate the issue.

Reply:
ok